### PR TITLE
Add Dockerfile template

### PR DESCRIPTION
**Pull request by yosifkit**
Also, sort all lists of things.

### CODE REVIEW

File: update.sh [hunk 1]
```diff
@@ -1,6 +1,16 @@
 #!/bin/bash
 set -e
 
+declare -A unsupportedModules
+unsupportedModules=(
```
Comment (tianon) [inline update.sh]: This should just be:

``` bash
declare -A unsupportedModules=(
    [...]...
```

File: update.sh [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+	[8.0]='memcached redis'
+)
+declare -A cmd
+cmd=(
```
Comment (tianon) [inline update.sh]: Samesies:

``` bash
declare -A cmd=(
...
```

Comment (yosifkit) [inline update.sh]: Both done.

**Comment by tianon** [comment]
LGTM

**Comment by pierreozoux** [comment]
Nice \o/

**Pull request merged**